openupgrade_scripts/scripts/website_forum/16.0.1.1/post-migration.py

```diff
@@ -0,0 +1,13 @@
+from openupgradelib import openupgrade
+from openupgradelib.openupgrade_160 import convert_string_bootstrap_4to5
+
+
+def _bs5_field_migration(env):
+    forums = env["forum.forum"].with_context(active_test=False).search([])
+    for forum in forums:
+        forum.faq = convert_string_bootstrap_4to5(forum.faq)
```

Comment:
@duong77476 có cần làm tổng quát không em? kiểu như tìm tất cả các trường kiểu html, và cho qua hàm `convert_string_bootstrap_4to5`?
```suggestion
        new_faq = convert_string_bootstrap_4to5(forum.faq)
        if forum.faq != new_faq:
            forum.faq = new_faq
```

Reply:
Nhưng cần lấy giá trị của trường thi có cách nào tổng quát không ạ

Reply:
@duong77476 anh merge trước, về cơ bản thì anh thấy vẫn cần làm tổng quát để tránh sót
em thử theo hướng như này nhé
xử lý ở end-migration:
1. select tất cả model, tên field từ bảng `ir.model.fields` có `store=True and ttype='html'`
2. sau đó search_read tất cả các model tìm được ở trên, đọc các fields ở trên với `domain=[]`
3. convert và write nó nếu có khác biệt về giá trị

Reply:
@royleviindoo Vâng a, tạm thời e tạo issue ở : https://github.com/Viindoo/OpenUpgrade/issues/537 xử sau ạ